test/parallel/test-repl-unsafe-array-iteration.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const { spawn } = require('child_process');
+
+function run(input, expectation) {
+  const node = spawn(process.argv0);
+
+  node.stderr.on('data', common.mustCall((data) => {
+    assert.ok(data.includes(expectation));
+  }));
```

Comment:
Actually we don't want anything printed to `stderr`, and we want the process to exit with a non-zero exit code.
```suggestion
  node.stderr.on('data', common.mustNotCall());
  node.on('exit', common.mustCall((code) => assert.strictEqual(code, 0)));
```